tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -191,6 +188,16 @@ func (s *stresser) Stress() error {
 	}
 	s.stressTable = createStressTable(stressEntries)
 
+	return nil
+}
+
+func (s *stresser) Stress() error {
+	if err := s.Setup(); err != nil {
```

Comment:
so is `Setup` for first time initialization or not? it obviously has no underlying contract if it's being used both ways